app/editor/src/store/hooks/admin/useActions.ts

```diff
@@ -44,28 +45,29 @@ export const useActions = (): [IAdminState, IActionController] => {
       },
       addAction: async (model: IActionModel) => {
         const response = await dispatch<IActionModel>('add-tag', () => api.addAction(model));
-        store.storeActions([...state.actions, response.data]);
+        store.storeActions((actions) => [...actions, response.data]);
+        await lookup.getLookups();
```

Comment:
Add this to any redux change that should also update the lookups.  This replaces the logic that was only partially supported by the old useLookupSync hook.